guides/release/components-update/html-is-html.md

```diff
@@ -0,0 +1,214 @@
+    musical scores.
+  </li>
+</ol>
+
+<p>
+  So please can you provide more information on each of these subjects,
+  including how long you'd expect the research to take, required staff and other
+  resources, and anything else you think we'd need to know? Thanks.
+</p>
+
+<h2>Exotic dance moves</h2>
+
+<p>
+  Yes, you are right! As part of my post-doctorate work, I <em>did</em> study
+  exotic tribal dances. To answer your question, my favourite dances are as
+  follows, with definitions:
+</p>
+
+<dl>
+  <dt>Polynesian chicken dance</dt>
+  <dd>
+    A little known but <em>very</em> influential dance dating back as far as
+    300<abbr title="Before Christ">BC</abbr>, a whole village would dance around
+    in a circle like chickens, to encourage their livestock or be "fruitful".
+  </dd>
+  <dt>Icelandic brownian shuffle</dt>
+  <dd>
+    Before the Icelanders developed fire as a means of getting warm, they used
+    to practice this dance, which involved huddling close together in a circle
+    on the floor, and shuffling their bodies around in imperceptably tiny, very
+    rapid movements. One of my fellow students used to say that he thought this
+    dance inspired modern styles such as Twerking.
+  </dd>
+  <dt>Arctic robot dance</dt>
+  <dd>
+    An interesting example of historic misinformation, English explorers in the
+    1960s believed to have discovered a new dance style characterised by
+    "robotic", stilted movements, being practiced by inhabitants of Northern
+    Alaska and Canada. Later on however it was discovered that they were just
+    moving like this because they were really cold.
+  </dd>
+</dl>
+
+<p>
+  For more of my research, see my
+  <a href="htp://www.example.com" title="Dr Gaye's exotic dance research"
+    >exotic dance research page</a
+  >.
+</p>
+
+<p>Yours sincerely,</p>
+
+<p>Dr Eleanor Gaye</p>
+
+<p>
+  University of Awesome motto: <q>Be excellent to each other.</q> --
+  <cite>The memoirs of Bill S Preston, <abbr title="Esquire">Esq</abbr></cite>
+</p>
+```
+
+We can also take the CSS from the MDN example and put it into our application, and the letter will be styled correctly.
+
+```css {data-filename=styles/app.css}
+body {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.sender-column {
+  text-align: right;
+}
+
+h1 {
+  font-size: 1.5em;
+}
+
+h2 {
+  font-size: 1.3em;
+}
+
+p,
+ul,
+ol,
+dl,
+address {
+  font-size: 1.1em;
+}
+
+p,
+li,
+dd,
+dt,
+address {
+  line-height: 1.5;
+}
+```
+
+You start building parts of an Ember application using HTML, so if you already know HTML and CSS, you know how to build a basic Ember application!
+
+You can even use SVG or web components without any changes. As long as your HTML is valid, Ember will render it.
+
+# Self-Closing Tags
+
+In addition to normal HTML syntax, Ember allows you to use self-closing syntax (`<div />`) as a shorthand for an opening and closing tag (`<div></div>`).
```

Comment:
Could we cut this section? I am not sure that it adds anything.

Reply:
I added it here because it comes up a lot in practical Ember and is in conflict with advice given everywhere else on the Internet. My intent was for this section to give an existing HTML developer the context they need to understand idiomatic Ember templates, and this is the one divergence from good HTML practice, so it felt important to call out.